### PR TITLE
docs(pagination): update a11y tab bugs for pagination nav

### LINKED ISSUE #4351: [Pagination] Fixes in a11y tab images
### Acceptance criteria

Some of the images in the a11y tab of Pagination are incorrect and need fixing.

<br>

1. [Labeling section](https://carbondesignsystem.com/components/pagination/accessibility/#labeling): Update the Select component's tooltip to read as "Page" instead of "Page number, of 40 pages"
Image

2. Tab stops will help navigate through the icon-only buttons. The "ellipse" which is a Select component, can be opened using space and arrow keys, navigated using arrow keys, selected using space and enter keys. Reflect this information in the image.
Image

3. Space and Enter selects, update the image reflecting the same. 
Image

### PR DESCRIPTION
**Pull request by laurenmrice**
Closes #4351 

This PR updates a few image bugs and content sections to reflect accurate accessibility information in regards to the pagination navs interactions. 

-----

**Changed**

- Changed images with correct annotations.
- Changed some sections of content to reflect correct annotations.

### CODE REVIEW

**Review by alisonjoseph — APPROVED** [review]

**Review by tay1orjones — APPROVED** [review]

File: src/pages/components/pagination/accessibility.mdx [hunk 1]
```diff
@@ -45,11 +45,8 @@ other accessibility considerations, some of which are described below.
 #### Pagination
 
 The tab order goes from left to right through the controls in the pagination
-variant. On focus, the dropdowns are opened with `Space` or with up or down
-arrows, which also cycle through the values. Both `Space` and `Enter` select a
-value and close the dropdown. The dropdown can also be closed by pressing `Esc`.
-The previous and next page arrow buttons are activated by `Space` or `Enter`
-keys.
+variant. On focus, the selects are opened with `Space` or with
+'Up`or`Down`arrows, which also cycle through the values. Both`Space`and`Enter`select a value and close the select. The select can also be closed by pressing`Esc`. The previous and next page arrow buttons are activated by pressing `Space`or`Enter`.
```
Comment (thyhmdo) [inline src/pages/components/pagination/accessibility.mdx:49]: `Up`or `Down` arrows, which also cycle through the values. Both`Space` and `Enter` select a value and close the select. The select can also be closed by pressing `Esc`. The previous and next page arrow buttons are activated by pressing `Space` or `Enter`.

File: src/pages/components/pagination/accessibility.mdx [hunk 2]
```diff
@@ -103,18 +100,20 @@ selection. When the previous and next page arrow buttons have focus, both
...
 <Column colLg={12}>
 
-![The menu opens by pressing Space and pressing Tab or Enter selects a page from the menu and closes it.](images/pagination-a11y-4.png)
+![The menu opens by pressing Space or Up or Down arrows and pressing Space or Enter selects a page from
+  the select menu and closes it.](images/pagination-a11y-4.png)
 
 <Caption>
-  The menu opens by pressing Space and pressing Tab or Enter selects a page from
-  the menu and closes it.
+  The menu opens by pressing Space or Up or Down arrows and pressing Space or
```
Comment (thyhmdo) [inline src/pages/components/pagination/accessibility.mdx:115]: Maybe "The menu opens by pressing Space, Up arrow, or Down arrow, and closes by pressing Space or Enter to select an item."

**Review by thyhmdo — COMMENTED** [review]
Looks great! just 2 comments

File: src/pages/components/pagination/accessibility.mdx (hunk 1 above)
Comment (thyhmdo) [inline src/pages/components/pagination/accessibility.mdx:49]:
```suggestion
`Up`or `Down` arrows, which also cycle through the values. Both`Space` and `Enter` select a value and close the select. The select can also be closed by pressing `Esc`. The previous and next page arrow buttons are activated by pressing `Space` or `Enter`.
```

**Comment by laurenmrice** [comment]
@thyhmdo Made the changes!

**Review by thyhmdo — APPROVED** [review]

**Pull request merged**